summary-visualiser/src/templates/partials/scenarioSummary.hbs

```diff
@@ -0,0 +1,34 @@
+<div class="scenario-summary">
+    <h1>
+        <span class="scenario-summary-title">{{title}}</span>
+        <span class="scenario-summary-run-id">(run {{data.run_id}})</span>
+    </h1>
+
+    <p class="scenario-summary-description">{{description}}</p>
+
+    <div class="scenario-summary-details">
+        <div class="scenario-summary-details-line">
+            <div class="scenario-summary-details-label">Started</div>
+            <div class="scenario-summary-details-value">{{datetime data.started_at}}</div>
+        </div>
+
+        <div class="scenario-summary-details-line">
+            <div class="scenario-summary-details-label">Duration</div>
+            <div class="scenario-summary-details-value">{{number data.run_duration}}s</div>
+        </div>
+
+        <div class="scenario-summary-details-line">
+            <div class="scenario-summary-details-label">Behaviours</div>
+            <div class="scenario-summary-details-value">
+                <ul class="scenario-summary-assigned-behaviours">
+                    {{#each data.assigned_behaviours}}<li><code>{{@key}}</code>: {{plural this " agent" " agents"}}</li>{{/each}}
```

Comment:
Styling of these looks a little funky:

<img width="313" height="53" alt="Image" src="https://github.com/user-attachments/assets/a33ef48e-ebcf-4a69-9232-72affa4f8bf6" />